onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.h

```diff
@@ -9,6 +9,9 @@
 #include "NvOnnxParser.h"
 #include "core/platform/ort_mutex.h"
 #include "tensorrt_execution_provider_info.h"
+#include <NvInferVersion.h>
```

Comment:
do you need to include these 3 headers?    aren't they needed in utils.h instead since you moved the GenerateId there?

Reply:
Thank you for pointing that out. Will move these dependencies back to util.h as well in the next revision